wiki/fr/fr-Software-Manual.md

```diff
@@ -0,0 +1,226 @@
+
+L'activation du mode stéréo augmentera le débit de votre flux. Assurez-vous que votre débit ne dépasse pas la bande passante disponible pour votre connexion internet.
+
+En mode stéréo, aucune sélection de canal audio pour l'effet de réverbération ne sera disponible dans la fenêtre principale puisque l'effet est alors appliqué aux deux canaux du signal stéréo dans ce cas.
+
+### Qualité audio
+
+![Qualité audio](https://user-images.githubusercontent.com/20726856/97362080-c42c3e80-18a0-11eb-87ac-9a4ca948bbcf.png)
+
+Plus la qualité audio est élevée, plus le débit de données de votre flux audio est élevé. Assurez-vous que votre débit ne dépasse pas la bande passante disponible pour votre connexion internet.
+
+### Niveau de nouveau client
+
+![Niveau de nouveau client](https://user-images.githubusercontent.com/20726856/97362173-e2923a00-18a0-11eb-8401-9ad71866f6b1.png)
+
+Ce paramètre définit le niveau du canal d'un client nouvellement connecté, en pourcentage. Si un nouveau musicien se connecte au serveur, il obtiendra le niveau initial spécifié si aucun autre niveau de canal provenant d'une connexion précédente de ce même client n'a déjà été enregistré. Vous pouvez configurer tous les clients d'un serveur occupé à ce niveau en utilisant Éditer > « Régler tous les charriots sur niveau d'un nouveau client ».
+
+### Thème graphique
+
+![Thème graphique](https://user-images.githubusercontent.com/20726856/97362214-f9389100-18a0-11eb-9adb-ebf2c1d18007.png)
+
+Il s'agit d'appliquer un thème graphique à la fenêtre principale.
+
+### Adresse personnalisée du serveur central
+
+![Adresse du serveur central](https://user-images.githubusercontent.com/20726856/97362265-0a819d80-18a1-11eb-9b0c-2f7ecf21841c.png)
+
+Laissez ce champ vide, sauf si vous devez entrer l'adresse d'un serveur central autre que celui par défaut.
+
+### Paramètre de l'état actuel de la connexion
+
+![Indicateurs de l'état de la connexion](https://user-images.githubusercontent.com/20726856/97362338-271dd580-18a1-11eb-99b7-02c9371a6258.png)
+
+Le temps de réponse est le temps nécessaire au flux audio pour faire l'aller-retour entre le client et le serveur.
+Ce délai est introduit par le réseau et devrait être d'environ 20 à 30 ms. Si le délai est supérieur à environ 50 ms, la distance qui vous sépare du serveur est trop importante ou votre connexion internet est trop mauvaise.
+
+Le délai global est calculé à partir du temps de réponse actuel et du délai introduit par les paramètres de la mémoire tampon.
+
+Le débit montant de l'audio dépend de la taille actuelle des paquets audio et du réglage de la qualité audio. Assurez-vous que le débit montant n'est pas supérieur à votre bande passante (vérifiez ça avec un service tel que [speedtest.net](http://speedtest.net)).
```

Comment:
* "_du réglage de la qualité audio_" -> "_du réglage de compression_"
* "_votre bande passante_" -> "_vitesse montante internet_"

Reply:
"Qualité audio" in the UI, "Compression" is is ambiguous in the audio/sound context.